pom.xml

```diff
@@ -739,6 +740,12 @@
                 <version>${xmpcore.version}</version>
             </dependency>
 
+            <dependency>
```

Comment:
No need for this - it is already defined in spring-boot parent.

Reply:
Fixed.